packages/ethereum/test/utils/time.ts

```diff
@@ -0,0 +1,34 @@
+import { ethers, providers } from 'ethers'
+
+export const advanceBlock = async (provider: providers.StaticJsonRpcProvider) => {
```

Comment:
Probably better to use the abstract provider than a concrete one such as `StaticJsonRpcProvider`

Reply:
This `advanceBlock` and other functions in this ts file only work in hardhat (or ganache to be precise), so `StaticJsonRpcProvider` is sufficient